tests/xcm-simulator/src/tests/fungible_assets.rs

```diff
@@ -538,12 +539,13 @@ fn receive_relay_asset_from_relay_and_send_them_back() {
     // Next step is to send some of relay native asset to parachain A.
     let withdraw_amount = 567;
     Relay::execute_with(|| {
-        assert_ok!(RelayChainPalletXcm::reserve_transfer_assets(
+        assert_ok!(RelayChainPalletXcm::limited_reserve_transfer_assets(
             relay_chain::RuntimeOrigin::signed(ALICE),
             Box::new(Parachain(1).into()),
             Box::new(alice.into()),
             Box::new((Here, withdraw_amount).into()),
             0,
+            WeightLimit::Unlimited,
```

Comment:
Super minor: `WeightLimit` enum is not already specify for *Unlimited* as "separator::" in this file. I like it for clarity, what about adding it everywhere?

Reply:
I haven't noticed that tbh - but let's keep it as `Unlimited` in this PR, the less changes the better at this point.

It's also more _polkadotish_ from what I've seen in the `polkadot-sdk` repo while investigating the `report_holding` issue.

Reply:
yes, I agree